### PR TITLE
style: Ensure minimum width for ArrivalByTimeChart 

### PR DESCRIPTION
**Pull request by R894**
# Description
Fixes ArrivalByTimeChart being too small on lower resolutions by adding a minimum width to the component

## Screenshots
Before:
image

After:
image

### CODE REVIEW

**Review by shootermv — APPROVED** [review]
nice catch!

File: src/pages/dashboard/ArrivalByTimeChart/ArrivalByTimeChart.tsx [hunk 1]
```diff
@@ -51,7 +51,7 @@ export default function ArrivalByTimeChart({
         .map((group) => (
           <div key={group[0].name}>
             <h3 className="title">{group[0].name}</h3>
-            <ResponsiveContainer debounce={1000} width="100%" height={300}>
+            <ResponsiveContainer debounce={1000} width="100%" minWidth={300} height={300}>
```
Comment (NoamGaash) [inline src/pages/dashboard/ArrivalByTimeChart/ArrivalByTimeChart.tsx:54]: what's if the screen is narrower, or wider? I'm just not sure `minWidth` is the right solution for all devices

Comment (R894) [reply src/pages/dashboard/ArrivalByTimeChart/ArrivalByTimeChart.tsx]: using negative left margin on the component seems to have a similar effect, what do you think?
image

Comment (NoamGaash) [reply src/pages/dashboard/ArrivalByTimeChart/ArrivalByTimeChart.tsx]: negative margin is also a bad practice
I think [this thread](https://stackoverflow.com/questions/50891591/recharts-responsive-container-does-not-resize-correctly-in-flexbox) worth investigation - it seems like a very similar problem

Comment (R894) [reply src/pages/dashboard/ArrivalByTimeChart/ArrivalByTimeChart.tsx]: I've looked into it and after experimenting the solution that worked for me is updating the package and setting a right margin of 35, but I'm still not sure if that's the best solution

**Review by NoamGaash — APPROVED** [review]
Cool!

File: src/pages/dashboard/ArrivalByTimeChart/ArrivalByTimeChart.tsx (hunk 1 above)
Comment (NoamGaash) [reply src/pages/dashboard/ArrivalByTimeChart/ArrivalByTimeChart.tsx]: That's far better! Thank you for doing this research

**Pull request merged**